untitled.js

```diff
@@ -0,0 +1,14 @@
+var name = 'Christian'
```

Comment:
Remember we don't want to use `var` anymore. Also, would be good to be consistent with using `;` at the end of statements.